delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/CompositeViews.scala

```diff
@@ -96,11 +81,11 @@ final class CompositeViews private (
     * @param caller
     *   the caller that initiated the action
     */
-  def create(project: ProjectRef, source: Json)(implicit caller: Caller): IO[CompositeViewRejection, ViewResource] = {
+  def create(project: ProjectRef, source: Json)(implicit caller: Caller): IO[ViewResource] = {
     for {
-      pc           <- fetchContext.onCreate(project)
-      (iri, value) <- sourceDecoder(project, pc, source)
-      res          <- eval(CreateCompositeView(iri, project, value, source.removeAllKeys("token"), caller.subject, pc.base))
+      pc           <- toCatsIO(fetchContext.onCreate(project))
+      (iri, value) <- toCatsIO(sourceDecoder(project, pc, source))
+      res          <- eval(CreateCompositeView(iri, project, value, source, caller.subject, pc.base))
```

Comment:
Just want to check that the `removeAllKeys("token")` is correct to remove?

Reply:
Well, there is no token anymore in the payload, this was just a way so as not to save it unencrypted.